README-Dutch

```diff
@@ -39,48 +39,50 @@ De introductie van _The Turing Way_ begint vanaf 5:13.
 
 ### Over het  project
 
-Reproducible research is necessary to ensure that scientific work can be trusted.
-Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code.
-The goal is to ensure that all results can be independently verified and built upon in future work.
-This is sometimes easier said than done.
-Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. As these activities are not commonly taught, we recognise that the burden of requirement and new skill acquisition can be intimidating to individuals who are new to this world.
-*The Turing Way* is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do" even for people who have never worked in this way before.
-It will include training material on version control, analysis testing, and open and transparent communication with future users, and build on Turing Institute case studies and workshops.
-This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: <https://github.com/alan-turing-institute/the-turing-way>.
+Reproduceerbaar onderzoek is nodig om ervoor te zorgen dat wetenschappelijk werk kan worden vertrouwd.
+Financiers en uitgevers beginnen te eisen dat publicaties de onderliggende data en de analysecode bevatten.
+Het doel van reproduceerbaar onderzoek is om ervoor te zorgen dat alle resultaten onafhankelijk kunnen worden geverifieerd en gebruikt kunnen worden in toekomstig onderzoek.
+Dit is soms makkelijker gezegd dan gedaan.
+Het delen van deze onderzoeksresultaten betekent inzicht in gegevensbeheer, bibliotheekwetenschappen, softwareontwikkeling en continue integratietechnieken: vaardigheden die niet algemeen worden onderwezen of verwacht van academische onderzoekers en datawetenschappers. Aangezien deze activiteiten niet algemeen worden onderwezen, erkennen we dat de last van vereisten en het verwerven van nieuwe vaardigheden intimiderend kan zijn voor personen die nieuw zijn in deze wereld.
+* The Turing Way * is een handboek om studenten, hun supervisors, financiers en tijdschriftenredacteuren te ondersteunen om ervoor te zorgen dat reproduceerbare datawetenschap "te gemakkelijk is om niet te doen", zelfs voor mensen die nog nooit op deze manier hebben gewerkt.
```

Comment:
The term "supervisors" is used in Dutch day to day language, but isn't a Dutch word. I'm not sure if we have a word for this in Dutch, but somethings that come close are "opzichter" (not really used a lot) or in this context maybe "begeleider"?

Reply:
Thanks! I went with begeleider as that is a good Dutch substitute indeed :)